Makefile

```diff
@@ -1,14 +1,10 @@
 ARCHS = arm64 arm64e
-TARGET = iphone:clang:12.2:12.2
-DEBUG = 0
-THEOS_DEVICE_IP = 192.168.1.21
-
+TARGET = iphone::12.0:latest
```

Comment:
platform:compiler:target:SDK

The compiler is clang by default.

Reply:
In theos docs, it says
`platform:compiler:sdk_version:deployment_version`
My concern is I don't have sdk_version 12.2 and it's still working

Reply:
Yeah, this always confuses me.

Reply:
I finally figured out, if `sdk_version` in makefile doesn't match any in your `$THEOS/sdks` folder, it will automatically select latest version in that sdks folder ;) 
In my case I have 12.2 and 13.2, and makefile is 12.0, then sdk version to compile is 13.2
FYI